src/frequenz/sdk/actor/_power_managing/_power_managing_actor.py

```diff
@@ -133,17 +133,22 @@ def _add_bounds_tracker(self, component_ids: frozenset[int]) -> None:
             microgrid,
         )
 
-        if self._component_category is not ComponentCategory.BATTERY:
+        bounds_receiver: Receiver[SystemBounds]
+        # pylint: disable=protected-access
+        if self._component_category is ComponentCategory.BATTERY:
+            battery_pool = microgrid.battery_pool(component_ids)
+            bounds_receiver = battery_pool._system_power_bounds.new_receiver()
+        elif self._component_category is ComponentCategory.EV_CHARGER:
+            ev_charger_pool = microgrid.ev_charger_pool(component_ids)
+            bounds_receiver = ev_charger_pool._system_power_bounds.new_receiver()
+        # pylint: enable=protected-access
```

Comment:
They are part of the `BatteryPool` because there were there originally and we repurposed it for internal use.  And I followed the same pattern here too.  We need to investigate how to pull it out, because it is also part of the feedback loop of: its values are used to set power and set power results affect its values, which is already setup of the *pools.  

And their values are used by the PowerManager which sits in between.  So it would be some effort to move them out.  I'll make an issue.

Reply:
Marked it as post-1.0: https://github.com/frequenz-floss/frequenz-sdk-python/issues/907